packages/config/config/devices/0x0154/flow_stop.json

```diff
@@ -0,0 +1,26 @@
+// Popp & Co Flow Stop
+// Version 1
+{
+	"manufacturer": "Popp & Co",
+	"manufacturerId": "0x0154",
+	"label": "Flow Stop",
+	"description": "",
```

Comment:
```suggestion
```

Reply:
Lint failed doing this?

Reply:
Let me check

Reply:
You're right. The description is enforced, but the style guide mentions it should be omitted - that's inconsistent. Empty description is bad though 😅

Reply:
Let's make the description `"Valve Actuator"` then.

Reply:
Should I just prefix the description for now, or you going to tackle the linter?